--- syslinux.cfg.orig	2025-10-28 13:04:34.367222073 +0100
+++ syslinux.cfg	2025-10-28 11:55:44.441668850 +0100
@@ -2,6 +2,7 @@
 # search path for the c32 support libraries (libcom32, libutil etc.)
 path debian-installer/amd64/boot-screens/
 prompt 0
-timeout 0
+timeout 100
+totaltimeout 600
 include debian-installer/amd64/boot-screens/menu.cfg
 default debian-installer/amd64/boot-screens/vesamenu.c32
